src/components/elements/table/TableRowActions.tsx

```diff
@@ -0,0 +1,63 @@
+import { Stack } from '@mui/material';
+import { useMemo } from 'react';
+import ButtonLink from '../ButtonLink';
+import CommonMenuButton, { CommonMenuItem } from '../CommonMenuButton';
+
+export type TableRowActionsType = {
+  primaryAction?: CommonMenuItem;
+  secondaryActions?: CommonMenuItem[];
+};
+
+interface TableRowActionsProps<T> {
+  record: T;
+  recordName?: string;
+  getActions: (record: T) => TableRowActionsType;
+}
+
+const TableRowActions = <T extends { id: string }>({
+  record,
+  recordName,
+  getActions,
+}: TableRowActionsProps<T>) => {
+  const accessibleName = useMemo(
+    () => recordName || record.id,
+    [record.id, recordName]
+  );
+
+  const { primaryAction, secondaryActions } = useMemo(
+    () => getActions(record),
+    [getActions, record]
+  );
+
+  return (
+    <Stack direction='row' alignItems='center' gap={0.5}>
```

Comment:
```suggestion
    <Stack direction='row' alignItems='center' justifyContent='end' gap={0.5}>
```

I believe this should be right-aligned
<img width="1061" alt="Screenshot 2024-12-20 at 1 03 45 PM" src="https://github.com/user-attachments/assets/dec87b08-275a-4b59-adee-f849899b6eaf" />